sdks/python/container/py38/base_image_requirements.txt

```diff
@@ -98,12 +99,13 @@ pandas==1.3.4
 parameterized==0.7.5
 pbr==5.7.0
 Pillow==7.2.0
+pkg_resources==0.0.0
```

Comment:
this looks like a bug.. 
see:
https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command

sounds like we might need to grep it out from the output for the script to work from the local env? or perhaps this will also go away with virtualenv->venv switch, although I don't fully understand when virtualenv was involved here. FWIW I don't get this line when I run this script.

Reply:
Good catch. I can add a patch to grep out pkg_resources and remove it from this PR.

As far as why this happened.. I don't think virtualenv was ever involved. It's notable that this only happened for py38, and not the other python versions. I'm not sure what that means though.

Reply:
are you running it on mac?

Reply:
No I'm running on Debian (gLinux). But `./gradlew` was actually executed inside the `./start-build-env.sh` (ubuntu) container.

Added 99f489f to filter out pkg_resources